src/libraries/System.IO.FileSystem.Watcher/tests/FileSystemWatcher.Directory.Changed.cs

```diff
@@ -28,11 +31,11 @@ public void FileSystemWatcher_Directory_Changed_WatchedFolder()
         {
             using (var testDirectory = new TempDirectory(GetTestFilePath()))
             using (var dir = new TempDirectory(Path.Combine(testDirectory.Path, "dir")))
-            using (var watcher = new FileSystemWatcher(dir.Path, "*"))
+            using (var watcher = new FileSystemWatcher(GetDirectoryOrSymLink(dir.Path), "*"))
             {
                 Action action = () => Directory.SetLastWriteTime(dir.Path, DateTime.Now + TimeSpan.FromSeconds(10));
 
-                ExpectEvent(watcher, 0, action, expectedPath: dir.Path);
+                ExpectEvent(watcher, 0, action, expectedPath: null);
```

Comment:
I assume you changed this to null because there are zero events expected?

Reply:
Yup, I think this way is more clear.